dotcom-rendering/src/components/PrivacySettingsLink.importable.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+import { cmp, onConsent } from '@guardian/consent-management-platform';
+import type { Framework } from '@guardian/consent-management-platform/dist/types';
+import { palette } from '@guardian/source-foundations';
+import { useState } from 'react';
+import { useOnce } from '../lib/useOnce';
+
+const footerLink = css`
+	color: inherit;
+	text-decoration: none;
+	padding-bottom: 12px;
+	display: block;
+	line-height: 19px;
+
+	:hover {
+		text-decoration: underline;
+		color: ${palette.brandAlt[400]};
+	}
+`;
+
+type Props = {
+	extraClasses?: string;
+};
+
+export const PrivacySettingsLink = ({ extraClasses }: Props) => {
+	const [framework, setFramework] = useState<Framework>();
+
+	useOnce(() => {
+		void onConsent().then((consentState) => {
+			setFramework(consentState.framework ?? undefined);
+		});
+	}, []);
+
+	if (!framework)
+		return (
+			<span css={footerLink} style={{ color: 'transparent' }}>
+				…
```

Comment:
if you just want a place holder, would nothing be better than something could look meaningful until you decided it wasn't?

```suggestion
				&nbsp;
```